### ISSUE TITLE
Server download tool should use mods/files from local cache

### ISSUE THREAD

**Issue by LatvianModder** [CreeperHost/modpacklauncher#4]
If it doesn't already, it's not really clear, since it see Downloading for every file

**Comment by ThePaul-T** [comment]
Server downloader does not have a local cache at this time, not even 100% sure it's a good idea yet, at least not with the same implementation as the client one.

**Comment by LatvianModder** [comment]
Oh right, I meant the same cache as client, but I guess that would mean the tool would have to know where app is installed. Hmm.

**Comment by ThePaul-T** [comment]
Yeah, that'd be so many levels of buggy, that is almost certainly not going to happen (It might, but the complexity with cross platform etc, it'll be a much later feature)

**Comment by MarcoCaspers** [comment]
Server downloader tool for Linux should be a shellscript rather than the extremely buggy binary blob that it is now.

**Comment by ThePaul-T** [comment]
@Grolimo 
"Extremely buggy" yet one single issue open about it (That doesn't have a lot of useful information like which files), thanks for your constructive input.

The API is open to anyone, feel free to write a shell script that does threaded downloading over HTTP/2 (HTTP/3 as soon as possible) with checksum checking etc, but I am not doing it.

A none threaded one which works on only Unix based OS's will be as easy as using jq to loop through the files array and curl the files, doesn't need an official solution for that.

Oh, remember to make sure it works on windows too, you won't have curl etc there.

Do not go to other issues and randomly hijack them with useless input.
